src/api/spec/bootstrap/features/webui/projects/comments_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'browser_helper'
+
+RSpec.feature 'Comments', type: :feature, js: true do
+  let!(:user) { create(:confirmed_user) }
+
+  context 'when logged-in as a confirmed user' do
+    before do
+      login user
+    end
+
+    let!(:comment) { create(:comment_project, commentable: user.home_project, user: user) }
+
+    scenario 'can be answered' do
+      visit project_show_path(user.home_project)
+      find("#reply_link_id_#{comment.id}").click
```

Comment:
I think this is the wrong approach, the button should be a button instead of a span.

Reply:
This won't change this line if it's a `button` or a `span`

Reply:
It will! The reason  why `click('Reply') doesn't find it, is that it is not a `button`. We don't need to use the complicated `id`, but to make that what looks like a button is a button. :bowtie:

Reply:
It's already a [button](https://github.com/openSUSE/open-build-service/blob/1fdd9038a0316c7cd6497b96dd753062258780fc/src/api/app/views/webui2/webui/comment/_links.html.haml#L4-L5) anyway. We can try with `Reply` as you suggest

Reply:
I see this in the browser:

```
<span class="btn btn-sm btn-outline-secondary" id="reply_link_id_815556">
Reply
</span>
```

Reply:
You don't have the latest version of master.